Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN yum install -y redis \
     gcc-c++ \
     npm \
     wget \
-    totem
+    totem \
+    supervisor
```

Comment:
The recommended way to install supervisord is via pypi according to their docs. You can do that directly in this dockerfile though to save on putting the dep in requirements. If the version via yum is recent enough maybe that won't matter?